[FEATURE] Change the log level when an authentication error occurs

**Is your feature request related to a problem? Please describe.**

- Currently, when an authentication error occurs, the following log is output.

```
[2021-07-14 05:58:13 +0000] [AUTH-LOG] [4833] [ERROR] [172.16.239.3] [0x05B2377c63A8826db2117314b38710646C500434] password mismatch [in /app/ibet-Prime/app/log.py:100]
```

- The log level is set to ERROR, but authentication errors are not something that the system administrator needs to deal with immediately.

**Describe the solution you'd like**

- Change the log level to WARNING.

--- a/app/log.py
+++ b/app/log.py
@@ -97,7 +97,7 @@ def auth_info(req: Request, address: str, msg: str):
 
 
 def auth_error(req: Request, address: str, msg: str):
-    AUTH_LOG.error(__auth_format(req, address, msg))
+    AUTH_LOG.warning(__auth_format(req, address, msg))
 
 
 def output_access_log(req: Request, res: Response):

